src/pages/Checkout/CheckoutBetaPage.vue

```diff
@@ -469,13 +469,13 @@ export default {
 			this.apollo.mutate({
 				mutation: shopAddOnePrintKivaCard,
 			}).then(({ data }) => {
-				this.setUpdatingTotals(false);
 				if (!data.shop || !data.shop.addOnePrintKivaCard) {
 					console.error(data);
+					this.setUpdatingTotals(false);
 					return;
 				}
 				this.$kvTrackEvent('promo', 'click', 'BasketKCUpsell');
-				this.refreshTotals();
+				window.location.reload();
```

Comment:
I saw this line and it set off an alarm in my head 🚨. I tapped @mcstover on the shoulder and he explained we'll just have this hard page reload in theretemporarilyy until we figure out how to refresh the basket to get newly added kiva cards to display right after they are added.